governance/votes/Executive vote - November 14, 2024.md

```diff
@@ -0,0 +1,216 @@
+
+- **Authorization**: [Governance Poll](https://vote.makerdao.com/polling/QmNTKFqG#poll-detail)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/14-nov-2024-proposed-changes-to-spark-for-upcoming-spell/25466)
+
+If this executive proposal passes, then the SparkLend D3M [Buffer (`buf`)](https://sky-atlas.powerhouse.io/A.3.8.1.3.1.4_Buffer_Definition/039d633f-bccf-492f-9c6d-c167e8b5a292|57eaf45219be9dbc3712) parameter on mainnet will be increased by 50 million DAI from 50 million DAI to **100 million DAI**.
+
+### Gelato Keeper Treasury Address Update
+
+- **Authorization**: [Forum Post](https://forum.sky.money/t/gelato-keeper-update/25456/3)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/gelato-keeper-update/25456)
+
+If this executive proposal passes, then the Gelato Keeper Treasury address will be updated to **[0x5041c60C75633F29DEb2AED79cB0A9ed79202415](https://etherscan.io/address/0x5041c60C75633F29DEb2AED79cB0A9ed79202415)**.
+
+### ConsolFreight (RWA-003) Debt Write-Off and Associated DAO Resolution Approval
+
+- **Authorization**: [Forum Post](https://forum.sky.money/t/consolfreight-rwa-003-cf4-drop-default/21745/22)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/consolfreight-rwa-003-cf4-drop-default/21745/21)
+
+If this executive proposal passes, then the following actions will take place.
+
+#### Debt Write-off
+
+If this executive proposal passes, the outstanding debt of the RWA-003 vault will be written off by calling `cull()` on the vault's oracle. Prior to this, `drip()` will be called on the Jug to ensure that all accrued interest is accounted for. This will reduce the Surplus Buffer to cover the outstanding debt of the vault.
+
+Please review this [forum post](https://forum.sky.money/t/consolfreight-rwa-003-cf4-drop-default/21745/21) from Steakhouse for the particulars of the events leading to this action.
+
+#### DAO Resolution Approval
+
+If this executive proposal passes, the DAO Resolution with the IPFS hash [QmX4DdVBiDBjLXYT4J4jC1XMdTn2Q7Ao8L66pKB8N3yETA](https://gateway.pinata.cloud/ipfs/QmX4DdVBiDBjLXYT4J4jC1XMdTn2Q7Ao8L66pKB8N3yETA) will be approved.
+
+This will authorize RWA Foundation to receive any recovered funds from ConsolFreight and to use them to fund operational expenses.
+
+### Facilitator Vesting Stream Creation
+
+- **Authorization**: [Atlas 1.6.2.4.1](https://sky-atlas.powerhouse.io/A.1.6.2.4.1_List_of_Facilitator_Budgets/c511460d-53df-47e9-a4a5-2e48a533315b|0db3343515519c4a)
+
+If this executive proposal passes, the following vesting streams will be created. 
+
+Note that DssVestSuckable for USDS and DSSVestTransferrable for SKY are not yet available. As a result, short streams for equivalent values of DAI and MKR are used. It is hoped that by the end of January, when these streams expire, it will be possible to switch the streams for these budgets to USDS and SKY, as detailed in The Atlas.
```

Comment:
```suggestion
Note that DssVestSuckable for USDS and DSSVestTransferrable for SKY are not yet available. As a result, short streams for equivalent values of DAI and MKR are used. It is hoped that by the end of January, when these streams expire, it will be possible to switch the streams for these budgets to USDS and SKY, as detailed in the Atlas.
```

Reply:
We used `The Atlas` in the poll today as it had been used in past copy. I don't have a strong opinion about either direction, but we should probably pick one for styling and stick to it.

Reply:
I don't mind either way, but if we use "The", we need to change it in other places where Atlas is mentioned within this Executive copy. Right now, every mention is with "the".

Reply:
>every mention is with "the".

There was only one mention using "the" which has been changed in https://github.com/makerdao/community/pull/2786/commits/373c79a2d11f98e8763eabcb5764d6014d50038d